util/patch/options.go

```diff
@@ -37,6 +37,13 @@ type HelperOptions struct {
 	// OwnedConditions defines condition types owned by the controller.
 	// In case of conflicts for the owned conditions, the patch helper will always use the value provided by the controller.
 	OwnedConditions []clusterv1.ConditionType
+
+	// LoggingEnabled enables the patch helper to log the content of the patch.
+	// Warning: This will log all keys and values from a patch including any sensitive information contained within.
+	LoggingEnabled bool
+
+	// SanitizedLoggingPaths will replace the values at each of the passed keys with "redacted".
+	SanitizedLoggingPaths []string
```

Comment:
What About
```suggestion
	PatchLoggingSanitizedPaths []string
```
To align with PatchLoggingEnabled